### DISCUSSION TITLE
Detox not printing number of tests passed or failed?

### DISCUSSION THREAD

**Discussion by saumya66** [wix/Detox#4121] [Q&A] [answered]
So recently we decided to go with Detox to do end to end testing of our react native app. Everything works fine but at the end we don't see the number of tests failed or passed.

The integration was done exactly as said in the documentation of detox. Let me know if you need to know anything else!

We see the final result like this but how many tests passed or failed does not get printed at the end. How can this be achieved? :

![](https://i.stack.imgur.com/vb7Ln.png)

**Comment by noomorph** [comment]
May I see your Jest config?

**Reply by saumya66** [reply]
Thanks for getting back! Sure here it is : 

```
/** @type {import('@jest/types').Config.InitialOptions} */
module.exports = {
  rootDir: '..',
  testMatch: ['<rootDir>/e2e/**/*.test.js'],
  testTimeout: 120000,
  maxWorkers: 1,
  globalSetup: 'detox/runners/jest/globalSetup',
  globalTeardown: 'detox/runners/jest/globalTeardown',
  reporters: ['detox/runners/jest/reporter'],
  testEnvironment: 'detox/runners/jest/testEnvironment',
  verbose: true,
};
```

**Reply by noomorph** [reply]
I'll check on my side today. I suspect you have an early exit for some reason, but I should check a few more things first.

**Reply by saumya66** [reply]
Thanks that will be super helpful!

**Reply by noomorph** [reply, marked as answer]
It took longer than expected to resolve, but I never lost sight of this bug report. @saumya66, this will be resolved in: https://github.com/wix/Detox/pull/4199

**Comment by asafkorem** [comment]
The fix for this issue was released with [v20.12.2](https://github.com/wix/Detox/releases/tag/20.12.2). @noomorph 👑
